macOS_Catalina.md

```diff
@@ -1,8 +1,8 @@
-# Installation notes for macOS Catalina (v10.15)
+# Installation notes for macOS Catalina (v10.15) Updates
```

Comment:
I don't think "Updates" is needed here, it's not _just_ about updates, it's about upgrades as well, so if we remove this word it still applies to both cases

Reply:
Sure, done. Happy to make changes given any guidance about wording, etc.